core/modules/system/system.module

```diff
@@ -2406,6 +2406,14 @@ function system_block_configure($delta, $settings) {
   if ($delta === 'header') {
     return system_header_block_form($settings);
   }
+  elseif ($delta === 'breadcrumb') {
+    $form['current'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Display the current page title as the last breadcrumb'),
+      '#default_value' => $settings['current'],
```

Comment:
`$settings['current']` may be not set the first time the block is configured, and this throws a PHP notice. We should make this be `!empty($settings['current'])`.